WordPress/Classes/ViewRelated/Stats/Extensions/Double+Stats.swift

```diff
@@ -1,8 +1,14 @@
 import Foundation
 
+fileprivate struct Unit {
+    let abbreviation: String
+    let name: String
+}
+
 extension Double {
 
     private static var _numberFormatter: NumberFormatter = NumberFormatter()
+    private static var _decimalFormatter: NumberFormatter = NumberFormatter()
```

Comment:
Is there a reason why we use `var` instead of `let`? Do we change these somewhere? I think I missed it. 🤔